tensorvision/train.py

```diff
@@ -103,9 +104,9 @@ def initialize_training_folder(hypes):
 
     # TODO: read more about loggers and make file logging neater.
 
-    hypes_file = os.path.basename(tf.app.flags.FLAGS.hypes)
-    _copy_parameters_to_traindir(
-        hypes, hypes_file, "hypes.json", target_dir)
+    target_file = os.path.join(target_dir, 'hypes.json')
```

Comment:
This is the kind of breaking change which I would expect to be in the commit message.